ci_framework/roles/devscripts/tasks/cleanup.yml

```diff
@@ -31,7 +31,7 @@
 - name: Cleanup the devscripts working directory.
   become: true
   ansible.builtin.file:
-    path: "{{ cifmw_devscripts_config.working_dir }}"
```

Comment:
nope, we want to `|combine` like it's done in the main.yml here: 
https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/devscripts/tasks/main.yml#L18-L25